### PR TITLE
SAK-42353 - Better handling of unlinked tools

### PR DESCRIPTION
**Pull request by csev**

### CODE REVIEW

File: basiclti/basiclti-tool/src/webapp/vm/lti_content_insert.vm [hunk 1]
```diff
@@ -23,32 +27,39 @@
...
 #end
-#if ( $formInput ) 
+#if ( $formInput )
 		$formInput
-#end
 <input #if($placement) disabled="disabled" checked="checked" #end type="checkbox" name="add_site_link"/> $tlang.getString("bl_addlink")
+#else
+#if ( $oldContentId )
+$tlang.getString("add.tool.select")
+<input name="id" type="hidden" value="$oldContentId">
```
[the remark this answers is not in the document]
Comment (csev) [reply basiclti/basiclti-tool/src/webapp/vm/lti_content_insert.vm]: It is a hidden field :)

File: basiclti/basiclti-tool/src/webapp/vm/lti_content_insert.vm [hunk 2]
```diff
@@ -23,32 +27,39 @@
...
+#end
+#end
                 <input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
 		<p class="act">
-			<input type="submit" accesskey ="s" class="active" name="$doAction" 
+			<input type="submit" accesskey ="s" class="active" name="$doAction"
 				value="$tlang.getString('gen.save')" />
+            <input type="submit" accesskey ="x" name="$doCancel" value="$tlang.getString('gen.cancel')"
+                 onclick="location = '$sakai_ActionURL.setPanel("ToolSite")';return false;">
```
[the remark this answers is not in the document]
Comment (ern) [reply basiclti/basiclti-tool/src/webapp/vm/lti_content_insert.vm]: I think this link shows you how it should be done https://dequeuniversity.com/rules/axe/3.3/input-button-name
from what I can tell this it is missing an id
`<input type="submit" accesskey ="x" id="customizeFrom-cancel" name="$doCancel" value="$tlang.getString('gen.cancel')">`

[the remark this answers is not in the document]
Comment (fostersdesign) [reply basiclti/basiclti-tool/src/webapp/vm/lti_content_insert.vm]: I don't understand why input[type="submit"] requires a label or an id.

[the remark this answers is not in the document]
Comment (csev) [reply basiclti/basiclti-tool/src/webapp/vm/lti_content_insert.vm]: From Matt:
Hi Dr. Chuck, the webaim advice [1] for submit and reset is that no additional information is needed.

I've seen accesslint will catch some things that I've missed but isn't 100% for sure.

[1] https://webaim.org/techniques/forms/controls#button

**Pull request merged**